JoinColony/colonyCDapp
# feat: add redo to streaming payment

## Description

- Changed kebab menu options for streaming payment

## Testing

*  Install streaming payment extension
* Create streaming payment action
* In completed action screen open kebab menu
* Check if redo action works fine

## TODO

- Add redo action to activity table

Resolves https://github.com/JoinColony/colonyCDapp/issues/3141
